packages/manager/modules/cloud-connect/src/cloud-connect.class.js

```diff
@@ -164,11 +164,16 @@ export default class CloudConnect {
   }
 
   setServiceKeys(serviceKeys) {
-    if (find(serviceKeys, { status: STATUS.ACTIVE })) {
-      set(this, 'validServiceKey', true);
-    } else {
-      set(this, 'validServiceKey', false);
-    }
+    set(this, 'validServiceKey', false);
+    map(serviceKeys, ({ status }) => {
+      if (
+        status === STATUS.ACTIVE ||
+        status === STATUS.DOING ||
+        status === STATUS.TO_CHECK
+      ) {
+        set(this, 'validServiceKey', true);
+      }
+    });
```

Comment:
Please avoid to use `lodash`
```suggestion
    this.validServiceKey = serviceKeys.some(({status}) => [STATUS.ACTIVE, STATUS.DOING, STATUS.TO_CHECK].includes(status));
    this.serviceKeys = serviceKeys;
```